packages/connect-react/src/components/CorbadoConnectProvider.tsx

```diff
@@ -29,7 +30,7 @@ const CorbadoConnectProvider: FC<PropsWithChildren<CorbadoConnectProviderProps>>
     >
       <ModalProvider>
         {!isWebJs && <CorbadoConnectModal />}
-        {children}
+        <div className={`light ${enableHighlight ? 'cb-highlight' : ''}`}>{children}</div>
```

Comment:
this will work for VR (so ok for now) but it's not going to work for someone who uses the react component (a react developer would wrap most of his code inside the CorbadoConnectProvider) => ok for now, because VR does not use React, but please add a comment that explains this problem